aviatrix/data_source_aviatrix_all_transit_gateways.go

```diff
@@ -0,0 +1,870 @@
+package aviatrix
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	//"github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix"
+	"github.com/AviatrixSystems/terraform-provider-aviatrix/v2/goaviatrix"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceAviatrixListAllTransitGateway() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAviatrixAllTransitGatewayRead,
+
+		Schema: map[string]*schema.Schema{
+			"all_transit_gateway": {
```

Comment:
I don't see why this attribute is needed. All it does is block the data source from reading, but I feel like the customer can just remove the data source to have the same effect.

Reply:
yes, I have remove this attribute, currently the data source is like:
data "aviatrix_all_transit_gateways" "foo" {

}


which doesn't need any attribute